docs/get-started/data-collection.md

```diff
@@ -165,6 +165,36 @@ Operator dispatch contains a hub-based resolve that loads operators from [towhee
 
 This section will explain how to use `DataCollection` to complete your daily data science works.
 
+## Image Search
+
+Preparing code and data:
+
+```python
+import towhee
+from towhee.functional import DataCollection
+
+dataset = DataCollection.from_zip('./image_dataset.zip', '*/dataset/*.JPEG').unstream()
```

Comment:
Maybe can change the zip path to URI(https://towhee.io/towhee/image-dataset/media/branch/main/image_dataset.zip)